src/SayMore/UI/ConvertMediaDlgViewModel.cs

```diff
@@ -38,6 +38,7 @@ public class ConvertMediaDlgViewModel
 		public ConvertMediaUIState ConversionState { get; private set; }
 		public MediaFileInfo MediaInfo { get; private set; }
 		public string OutputFileCreated { get; private set; }
+		public readonly string OutputFile;
```

Comment:
It doesn't look like this needs to be public. If it's going to be a private field, it should be named _outputFile. If it is needed publicly, it should be made a property (like the ones above it).

Reply:
Done.